use-timescale/data-tiering/about-data-tiering.md

```diff
@@ -9,45 +9,32 @@ cloud_ui:
 
-Apache Parquet allows for more efficient scans across longer time periods, and $CLOUD_LONG uses other metadata and query optimizations to reduce the amount of data that needs to be fetched from the object storage tier to satisfy a query.
+## High-performance storage 
 
-Regardless of where your data is stored, you can still query it with standard SQL. A single SQL query transparently pulls data from the appropriate chunks using the chunk exclusion algorithms. You can `JOIN` against tiered data, build views, and even define continuous aggregates on it. In fact, because the implementation of continuous aggregates also uses hypertables, they can be tiered to low-cost storage as well. 
+High-performance storage is where your data is stored by default, until you [enable tiered storage][manage-tiering] and start [moving data to the low-cost tier][move-data]. In the high-performance storage, your data is stored in the block format and optimized for frequent querying. The [$HYPERCORE row-columnar storage engine][hypercore] available in this tier is designed specifically for real-time analytics and can compress your data by up to 90%. Coupled with other optimizations, $CLOUD_LONG high-performance storage makes sure your data is always accessible and your queries run at lightning speed. 
 
-$COMPANY charges only for the storage that your data occupies in S3, regardless of whether it was compressed in $CLOUD_LONG before tiering. There are no additional expenses, such as data transfer or compute. 
+$CLOUD_LONG high-performance storage comes in two types—standard and enhanced. 
 
-## Benefits of the object storage tier
+- **Standard** (default): provides up to 16TB of storage and 16,000 IOPS. Based on [AWS EBS gp3][aws-gp3].
+- **Enhanced**: provides up to 64TB of storage and 64,000 IOPS. Based on [EBS io2][ebs-io2].
 
-The object storage tier is more than an archiving solution. It is also:
-
-*   **Cost-effective:** store high volumes of data at a lower cost.
-    You pay only for what you store, with no extra cost for queries.
-
-*   **Scalable:** scale past the restrictions imposed by storage that can be attached
-    directly to a Timescale service (currently 16 TB).
-
-*   **Online:** your data is always there and can be [queried when needed][querying-tiered-data]. 
-
-## Architecture
-
-The tiered storage backend works by periodically and asynchronously moving older chunks from the high-performance storage to the object storage. 
-There, it's stored in the Apache Parquet format, which is a compressed columnar format well-suited for S3. Within a Parquet file, a set of rows is grouped together to form a row group. Within a row group, values for a single column across multiple rows are stored together.
+[See the differences][aws-storage-types] in the underlying AWS storage. You [enable enhanced storage][enable-enhanced] as needed in $CONSOLE. 
```

Comment:
Hmmm, I would try and include see the differences in the list.

Reply:
@rahilsondhi I think you have had a table for this comparison - could you give me access to it, please?